docker/development/medusa-be/Dockerfile

```
@@ -15,19 +16,31 @@ WORKDIR /var/www
 FROM base AS dev
 ENV NODE_ENV=development
 ENV DATABASE_TYPE=postgres
+COPY package.json /tmp/corepack/package.json
+RUN cd /tmp/corepack && corepack install && rm -rf /tmp/corepack
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)